graph/src/task_spawn.rs

```diff
@@ -0,0 +1,31 @@
+use futures03::executor::block_on;
+use futures03::future::{FutureExt, TryFutureExt};
+use std::future::Future as Future03;
+use std::panic::AssertUnwindSafe;
+use tokio::task::JoinHandle;
+
+fn abort_on_panic<T: Send + 'static>(
+    f: impl Future03<Output = T> + Send + 'static,
+) -> impl Future03<Output = T> {
+    // We're crashing, unwind safety doesn't matter.
+    AssertUnwindSafe(f).catch_unwind().unwrap_or_else(|_| {
+        println!("Panic in tokio task, aborting!");
+        std::process::exit(1)
+    })
+}
+
+pub fn spawn<T: Send + 'static>(f: impl Future03<Output = T> + Send + 'static) -> JoinHandle<T> {
```

Comment:
It may be useful to add a comment that this aborts on panic. That may be unexpected, especially if tokio doesn't match that behavior.

Reply:
True, done.